WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -112,6 +123,57 @@ public static async Task<AllFeeEstimate> EstimateAllFeeAsync(this IRPCClient rpc
 				.ToDictionary(x => x.Blocks, x => (int)Math.Ceiling(x.FeeRate.SatoshiPerByte));
 		}
 
+		private static Dictionary<int, int> GetFeeEstimationsFromMempoolInfo(MemPoolInfo mempoolInfo)
+		{
+			const int BlockSize = 1_000_000;
+			static IEnumerable<(int Size, FeeRate From, FeeRate To)> SplitFeeGroupInBlocks(FeeRateGroup group)
+				=> Enumerable
+					.Range(0, (int)group.Sizes / BlockSize)
+					.Select(b => (Sizes: BlockSize, From: group.From, To: group.To));
+
+			// Filter those groups with very fee transactions (less than 1%).
```

Comment:
You wanted to say: "with very **high** fee transactions"?

Reply:
```suggestion
			// Filter those groups with very high fee transactions (less than 1%).
```

Reply:
Yes, thanks.